content/applications/general/email_communication/faq.rst

```diff
@@ -0,0 +1,174 @@
+instead of by emails. 
+
+SMTP Error
+**********
+
+You can find out why an email wasn't transmitted successfully by reviewing the Simple Mail 
+Transport Protocol (SMTP) error messages. SMTP is a protocol to describe the email structure 
+and transmit it over the Internet, and the error messages generated by email services are helpful 
+tools to diagnose and troubleshoot email problems.
+
+No Error
+********
+
+Odoo is not always capable of providing information for the reason it failed. The different 
+providers implement a personalized policy of the bounce emails and it is not always possible 
+for Odoo to interpret it correctly.
+
+If you have this problem on a recurring basis with the same client or the same domain, please 
+do not hesitate to contact `Odoo Support <https://www.odoo.com/help>`_ for help in finding a reason.
+
+Note: in such case, one of the most common reasons is related to :ref:`SPF <email_communication/spf_compliant>` 
+and/or :ref:`DKIM <email_communication/DKIM_compliant>` configuration.
+
+Why is my email sent late?
+**************************
+
+It may happen that you schedule an email campaign but it is not sent on time. We know that 
+we use a delayed job to send emails that we consider as not urgent (Newsletters concept 
+such as mass mailing, marketing automation, events). The system utility **cron** can be used 
+to schedule programs to run automatically at predetermined intervals. We use that policy in order 
+to avoid cluttering the mail servers and prioritize the communication. 
+
+The emails considered urgent (communication from one person to another one such as 
+Sales Orders, Invoices, Purchase Orders, etc.) are sent directly. 
+
+.. image:: faq/email_scheduled_later.png
+   :align: center
+   :alt: Email scheduled to be sent later.
+
+By default, the Mass Mailing cron runs every 60 minutes. So, you should wait maximum an hour 
+before the campaign is actually sent.
+
+Incoming emails
+===============
+
+When you have an issue with incoming emails, there might not be an indication per se in Odoo. 
+This is the client who tries to contact a database who will get a bounce (most of the 
+time 550: mailbox unavailable).
+
+Emails are not received
+-----------------------
+
+Depending on the platform you are using:
+
+- The **Odoo.sh** users can find their live logs on the folder :file:`~/logs/`.
+
+- The folder :file:`~/logs/` (preferably accessed by the command line) of an Odoo.sh contains 
+  a list of files containing the logs of the database. The log files are created everyday 
+  at 5:00 AM UTC. The two last days are not compressed, while the older ones are, in order 
+  to gain space. The naming of the files for Today and Yesterday are odoo.log and odoo.log.1. 
+  For the following, they are named with their dates and compressed.
+  See the Odoo.sh documentation about :ref:`logs <odoosh/logs>`. 
+  Use the command ``grep`` and ``zgrep`` (for the compressed ones) to search through the files.
```

Comment:
Either
```suggestion
  to gain space. The naming of the files for Today and Yesterday are odoo.log and odoo.log.1. For the following, they are named with their dates and compressed. See the Odoo.sh documentation about :ref:`logs <odoosh/logs>`. Use the command ``grep`` and ``zgrep`` (for the compressed ones) to search through the files.
```
or
```suggestion
  to gain space. The naming of the files for Today and Yesterday are odoo.log and odoo.log.1. 
  
  For the following, they are named with their dates and compressed.
  
  See the Odoo.sh documentation about :ref:`logs <odoosh/logs>`. 
  
  Use the command ``grep`` and ``zgrep`` (for the compressed ones) to search through the files.
```
or a combination of the two, depending on whether the line break should be visible.